assets/css/style.css

```diff
@@ -70,6 +79,12 @@ body {
 
 #search-icon {
     width: 25px;
+    position: absolute;
+    width: 50%;
+    height: 50%;
+    margin-left: 15px;
+    margin-top: 15px;
+
 }
 
```

Comment:
I think the searchbar css should go here, as the dynamic html displays underneath it.